src/fragments/lib/auth/js/set-up-auth.mdx

```diff
@@ -1,4 +1,3 @@
-
 > *This guide provides essential information to help you select and complete the activities you need for your Amplify project. We organized each section around a distinct job or decision point to help you understand your available options, steps to complete, and recommended best practices. For users who want to learn more, we included extra details and explanations in expandable sections within the guide.*
 
 In this guide, you will learn how to set up Amplify Authentication. This includes setting up and connecting your backend resources, determining your integration path, and enabling sign-up, sign-in, and sign-out. We will also provide more context on how resources are managed and created with Amplify to help you make decisions, and understand any long-term impact of those decisions, for your use case.
```

Comment:
Small tweaks:

In this guide, you will learn how to set up Amplify **Auth**. This includes setting up and connecting your backend resources, determining your integration path, and enabling sign-up, sign-in, and sign-out. We will also provide more context on how resources are managed and created with Amplify to help you make decisions, and understand any long-term impact of those decisions.